pkg/zabbix/methods.go

```diff
@@ -173,7 +173,7 @@ func filterItemsByQuery(items []*Item, filter string) ([]*Item, error) {
 	for _, i := range items {
 		name := i.Name
 		if re != nil {
-			if re.MatchString(name) {
+			if match, err := re.MatchString(name); match && err != nil {
```

Comment:
That should be  like
```suggestion
			if match, err := re.MatchString(name); match && err == nil {
```